scripts/templates/trcddi.cpp.mako

```diff
@@ -49,6 +49,10 @@ namespace ur_tracing_layer
 
         context.notify_end(${th.make_func_etor(n, tags, obj)}, "${th.make_func_name(n, tags, obj)}", &params, &result, instance);
 
+        std::ostringstream args_str;
+        ur::extras::printFunctionParams(args_str, ${th.make_func_etor(n, tags, obj)}, &params);
+        context.logger.info("---> ${th.make_func_name(n, tags, obj)}({}) -> {};", args_str.str(), result);
```

Comment:
I think we might want to print just `---> func_name` before the function call, and then the rest `(args) -> result` after it has finished. I vaguely remember SYCL_PI_TRACE doing something like that. It's useful for debugging hangs.

Reply:
That makes sense, I've made that change